client-react/src/ApiHelpers/ArmHelper.ts

```diff
@@ -211,6 +216,187 @@ const MakeArmCall = async <T, U = T>(requestObject: ArmRequestObject<U>): Promis
   return retObj;
 };
 
+const pollForCompletion = <T, U = T>(response: ArmBatchObject, request: ArmRequestObject<U>) => {
+  const location = getHeader('location', response.headers);
+  const azureAsyncOperation = getHeader('Azure-AsyncOperation', response.headers);
+
+  if (location) {
+    return pollLocationForCompletion(response, location, request);
+  } else if (azureAsyncOperation) {
+    return pollAzureAsyncOperationForCompletion(response, azureAsyncOperation, request);
+  } else if ((<any>response.content)?.properties.provisioningState) {
+  } else {
```

Comment:
Is this block supposed to be empty? If so, consider doing a negative check instead:
```suggestion
  } else if (!(<any>response.content)?.properties.provisioningState) {
```

Reply:
I forgot to include the pollProvisioningState call here